environment.yml

```diff
@@ -0,0 +1,19 @@
+name: linsolve
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - nose
+  - numpy>=1.10
+  - pip
+  - astropy>=2.0
+  - h5py
+  - six
+  - pyuvdata
+  - scipy
+  - pytest
+  - scikit-learn # from hera_cal
+  - pip:
+    - git+https://github.com/HERA-Team/hera_cal
```

Comment:
`hera_cal` requires aipy and as a result the installation is failing on travis. I think you'll probably just have to add aipy back in here or wait for a `hera_cal` which make aipy auto-downloaded as a dependency through pip.

Reply:
also please add `pytest-cov` it is necessary for generating coverage reports.

Reply:
Added pytest-cov. I'll wait for the upstream fixes for `aipy`, rather than putting redundant stuff in these commits. ATM, `hera_cal` is waiting on `pyuvsim`... so yeah, a bit of a chain here haha.